### PR TITLE
Weekly redesign

### PR DESCRIPTION
**Pull request by codyjames**

### CODE REVIEW

File: lib/changelog_web/templates/news_issue/_item.html.eex [hunk 1]
```diff
@@ -1,11 +1,17 @@
-<div class="list-item">
-  <h2 class="list-item__title"><%= link(@item.headline, to: @item.url) %></h2>
+<div class="list_item">
+  <%# TODO: Add ability to use render_item_source(@conn, @item) %>
```
Comment (codyjames) [inline lib/changelog_web/templates/news_issue/_item.html.eex]: @jerodsanto Need help here when you have time.

File: lib/changelog_web/templates/news_issue/_item.html.eex [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+  <p class="list_item-source">Item Source</p>
+  <h2 class="list_item-title"><%= link(@item.headline, to: @item.url) %></h2>
   <%= @item.story |> md_to_html |> raw %>
-<%= if @item.image do %>
-  <p>
-    <%= link to: @item.url do %>
-      <img src="<%= NewsItemView.image_url(@item, :large) %>" alt="<%= @item.headline %>">
-    <% end %>
+  <p class="list_item-meta">
+    <%# TODO: How do I loop through topics? %>
```
Comment (codyjames) [inline lib/changelog_web/templates/news_issue/_item.html.eex]: @jerodsanto Here too!

**Comment by codyjames** [comment]
@jerodsanto @adamstac Go ahead and give this a spin and let me know if you seeing anything that needs fixed.

**Comment by jerodsanto** [comment]
LGTM but I'll let @adamstac use the fine-tooth comb because I don't have the Invision project and he has the eye for the details on it

**Comment by adamstac** [comment]
Looks good @codyjames - there are few details I'll ship updates to here in a bit. Is there anything major on your side in the CSS you're working on that I may cause conflict to?

**Comment by codyjames** [comment]
@adamstac Nope, just keeping the css simple and trying not to use anything too modern. It's fairly minimal in there.

**Comment by adamstac** [comment]
I spent time looking into webfonts because they have great support across the clients our readers use thanks to `@import` support now and Google Fonts. Self hosting webfonts for use in email is still bad though. I found a better heading alternative to Sana Sans than Helvetica or Arial.

Here are a few items that need done still:

- [ ] Add class `.news_item--sponsored` to `.news_item` div when it's a sponsored item ~> [here](https://github.com/thechangelog/changelog.com/blob/3785b8700cb67c71c09ebcf8195483e4d8ac6982/lib/changelog_web/templates/news_issue/_item.html.eex#L1)
- [ ] Add the sponsor image and name of the sponsor in the `.news_item-source` class like we do on the web
- [ ] Update the issue date in the `.header` to use the hacker format 2018-01-14 ~> [here](https://github.com/thechangelog/changelog.com/blob/3785b8700cb67c71c09ebcf8195483e4d8ac6982/lib/changelog_web/templates/news_issue/show.html.eex#L39)

**Comment by jerodsanto** [comment]
ok @adamstac feel free to tweak and merge whenever you're ready ✊

**Comment by adamstac** [comment]
LGTM! 🎉

**Pull request merged**